src/main/resources/templates/datewarehouse/etl-jobs-config.json

```diff
@@ -379,16 +907,37 @@
 		"sourcePath" : "log.sagebase.org/certifiedUserPassingRecords/",
 		"tableDescriptor": {
 			"name": "certifiedquizquestionrecords",
-			"columns": {
-				"response_id": "bigint",
-				"question_index": "bigint",
-				"is_correct": "boolean",
-				"stack": "string",
-				"instance": "string"
+			"description" : "This table contain records of the questions of the certification quiz.Each record in the table is question attempted by the user in the quiz.",
+			"columns":[{
+				"name": "response_id",
+				"type" : "bigint",
+				"comment" : "The identifier of the certification quiz taken by the user."
```

Comment:
What is the difference between response_id and instance?  Can that be made more clear?